server/main.py

```diff
@@ -40,6 +37,23 @@ async def add_service(request: web.Request):
     return web.json_response({'success': True}, status=200)
 
 
+async def receive_alert(request: web.Request):
+    try:
+        # Not using json here, because we want to send a link through email
+        notification_id = request.query['notification_id']
+        primary_admin = request.query['primary_admin']
+        # DOZRO: validation
+    except KeyError as e:
+        return web.json_response({'error': str(e)}, status=400)
+
+    try:
+        db_access.update_notification_response_status(notification_id, primary_admin, db_conn)
```

Comment:
As the `primary_admin` variable is a string we need to convert it to bool.
```suggestion
        db_access.update_notification_response_status(notification_id, primary_admin == 'True', db_conn)
```